calico-enterprise/network-policy/domain-based-policy.mdx

```diff
@@ -229,6 +229,13 @@ spec:
       destination:
         selector: color == 'red'
 ```
+## Staged DNS Policy
```

Comment:
As written, it sounds like we are covering up a bug and forcing users to set a flag unnecessarily. I suspect we just need to emphasize the reason for choosing to align staged and enforced policy. For example:

When you stage a DNS policy, the goal is to preview and test flows to ensure the policy is correctly allowing or blocking traffic before enforcement. When a DNS policy first tries to reach a specified domain address or networkset, there can be a delay in reaching the domain that can result in a “no match” in flow logs in troubleshooting. Although this condition does not occur in enforced policies, you may want to align flow log results between staged and enforced policy by changing the `FelixConfiguration` parameter from `DNSPolicyMode` to `DelayDNSResponse`.